test/unit_test/cli/test_cli.py

```diff
@@ -284,7 +282,7 @@ def test_quantize_command(mock_repo_exists, mock_tempdir, mock_run, algorithm_na
     mock_tempdir.return_value = tmpdir.resolve()
     mock_run.return_value = {}
 
-    workflow_output_dir = tmpdir / "output_model" / algorithm_name
+    workflow_output_dir = tmpdir / algorithm_name
```

Comment:
I don't think the algorithm name exists anymore? This was for the behavior where each pass flow saved it's own output model

Reply:
Good call. That's correct. Updated.